src/modules/partition/Config.cpp

```diff
@@ -46,6 +46,32 @@ Config::swapChoiceNames()
     return names;
 }
 
+const NamedEnumTable< Config::LuksGeneration >&
+Config::luksGenerationNames()
+{
+    static const NamedEnumTable< LuksGeneration > names { { QStringLiteral( "luks1" ), LuksGeneration::Luks1 },
+                                                          { QStringLiteral( "luks2" ), LuksGeneration::Luks2 } };
+
+    return names;
+}
+
+const QString
+Config::luksGenerationToFSName( Config::LuksGeneration luksGeneration )
+{
```

Comment:
Wouldn't this be simpler as `luksGenerationNames().find(luksGeneration)` ? The named enums are supposed to support lookups both forward and back.

Reply:
Unless I misunderstand, it's not the same thing. The goal of this function is to convert the enum into the File System string set inside the `GlobalStorage` object. I used the `.find` function in line `#355`.

Reply:
This function might be removed depending on the option you prefer in the comment below.